api/src/main/java/org/pmiops/workbench/actionaudit/auditors/UserServiceAuditAdapterImpl.java

```diff
@@ -76,4 +76,21 @@ public void fireAdministrativeBypassTime(
 
     actionAuditService.send(eventBuilder.build());
   }
+
+  @Override
+  public void fireAcknowledgeTermsOfService(DbUser targetUser, Integer termsOfServiceVersion) {
+    actionAuditService.send(
+        ActionAuditEvent.builder()
+            .timestamp(clock.millis())
+            .agentType(AgentType.USER)
+            .agentId(dbUserProvider.get().getUserId())
+            .agentEmailMaybe(dbUserProvider.get().getUsername())
+            .actionId(actionIdProvider.get())
+            .actionType(ActionType.EDIT)
+            .targetType(TargetType.ACCOUNT)
```

Comment:
If you don't mind, please tag me on PRs with new audit log entries, since we don't have a great migration story there.

The main thing I see here is that the action and target don't feel quite right: at the moment compliance docs and compliance acceptance aren't first-class concepts in the audit system. It would be slightly more idiomatic (and potentially help with audit queries), if you had something like 
```
AgentType.USER
ActionType.ACCEPT
TargetType.COMPLIANCE_DOCUMENT

// 2 property fields:
DocumentTargetProperty.VERSION,
DocumentTargetProperty.NAME,
```

Then the log entry for DUA acceptance would be almost identical to this code (and in fact we could bust out a separate `ComplianceDocumentAuditor` when we're ready to do that.

Reply:
Filed [this story](https://precisionmedicineinitiative.atlassian.net/browse/RW-4372) to add constraints to these object types to make the intent more clear (and assist in query-building tools and services for this dataset).